.github/workflows/validate_ansible_roles.yaml

```diff
@@ -0,0 +1,24 @@
+name: Validate Ansible Roles
+on: build
+jobs:
+  validate:
+    name: Validate Ansible Roles
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Deps
+        uses: mstksg/get-package@master
+        with:
+                apt-get: cmake ninja-build libopenscap8 libxml2-utils expat xsltproc python3-jinja2 python3-yaml ansible-lint
+      - name: Checkout
+        uses: actions/checkout@v1
```

Comment:
@isimluk why not use https://github.com/marketplace/actions/ansible-lint ?

Reply:
@isimluk on second thought.... could we just do https://github.com/marketplace/actions/ansible-molecule ? ansible-lint exists already in molecule. It may not be feasible or work to have Molecule, but would be really nice to have.

Reply:
my 2c @redhatrises, but `ansible-molecule` does a acceptable job on specific use cases and might work on this project but will fail as soon as some actions, like operational testing (restarting services etc), has to be performed.

I personally use the following and it will result in a 5/5 on Ansible Galaxy (https://galaxy.ansible.com/konstruktoid/hardening).

```
  if ! find ./ -type f -name '*.y*ml' ! -name '.*' -print0 | \
    xargs -0 ansible-lint; then
      echo 'ansible-lint failed.'
      exit 1
  fi

  if ! find ./ -type f -name '*.y*ml' ! -name '.*' -print0 | \
    xargs -0 yamllint -d "{extends: default, rules: {line-length: {level: warning}}}"; then
      echo 'yamllint failed.'
      exit 1
  fi
```